runtime_lib/libxaie/CMakeLists.txt

```diff
@@ -42,4 +42,4 @@ include_directories(
 
 add_library(xaiengine SHARED ${libsources})
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libxaiengine.so DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib)
\ No newline at end of file
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libxaiengine.so DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib/libxaie)
```

Comment:
This should match the name of the library: xaiengine or aie-rt if we choose to move to that repo

Reply:
Done. matched to xaiengine until aie-rt gets updated.